clients/go/admin/client.go

```diff
@@ -194,6 +200,7 @@ func initializeClients(ctx context.Context, cfg *Config, tokenCache pkce.TokenCa
 		}
 
 		if opt != nil {
+			cs.authOpt = opt
```

Comment:
This statement will be executed only once... if we call initializeClients again, the returned clientSet won't have it set...

You probably want to create another global variable for the authOption and then reference it... just like adminConnection

Reply:
I think you're right. Let me fix that.

Reply:
Fixed